etc/smashbox.conf.template

```diff
@@ -132,3 +132,9 @@ reset_server_log = False
 # used in a test run to allow for more users sharing at time
 #
 share_sets = 1
+
+# properties for defining directory depth and directory layout
+#
+dir_depth = 5
```

Comment:
I get that but would rather not make the change now ;)  The way smashbox is currently designed, the properties file is used for defining test properties.  We can create an issue to refactor the property handling in general if you want.